app/models/product_import/entry_validator.rb

```diff
@@ -189,6 +196,43 @@ def unit_fields_validation(entry)
                              error: I18n.t('admin.product_import.model.conditional_blank'))
     end
 
+    def is_numeric(value)
+      return true unless Float(value, exception: false).nil?
+    end
+
+    def price_validation(entry)
+      return if is_numeric(entry.price)
+
+      if empty_or_placeholder_value(entry.price)
+        mark_as_invalid(entry, attribute: 'price',
+                               error: I18n.t('admin.product_import.model.blank'))
+      else
+        mark_as_invalid(entry, attribute: 'price',
+                               error: I18n.t('admin.product_import.model.incorrect_value'))
```

Comment:
you can use a trinary  for the `error` attribute

Reply:
Good suggestion. Thank you.
Will make the suggested change in the next revision with other comments (if any).

Reply:
Could not use tertiary operator here as the code style needed it be in one line and when I did that, it triggered another linter error for line too long. But made changes to take away code repetition as suggested.